Assignment/solution_student/css/style.css

```diff
@@ -31,9 +30,19 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
   font-size: 16px;
-  background: white;
+  background: #f0efef;
+  font-family: 'Open Sans', sans-serif;
+  color: #033048;
+
+}
+
+h1 {
```

Comment:
Both of these font-family declarations on the h1 and p are not needed, because you declared the font family on the body, which is inherited by everything else — the only reason you'd need to declare them here is if they had a different font-family from the rest of the page.